apps/safe-claiming-app/src/components/steps/Status/usePendingTx.ts

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react"
+import { TransactionStatus } from "@gnosis.pm/safe-react-gateway-sdk"
+import { useSafeAppsSDK } from "@gnosis.pm/safe-apps-react-sdk"
+
+const POLL_INTERVAL = 3000
```

Comment:
The block time is 13-14 seconds, I wonder if we can increase the interval to remove some load from the backend

Reply:
I'll increase it to 10 seconds.